### PR TITLE
Tidy up SCSS variables and imports

### PR DESCRIPTION
**Pull request by camillevilla**
Addresses #66 

This PR is mostly SCSS clean up. 
- adds `bootstrap_variables.scss` to SCSS imports
- migrates `bootstrap_overrides.scss` and `searchworks_mixins.scss` from SearchWorks
- cuts down on unused color variables in `sul_variables.scss`

I'm not sure how useful the `.panel` styles are going to be. It looks like they were cut from [Bootstrap 4](https://v4-alpha.getbootstrap.com/migration/#panels-thumbnails-and-wells) and not-exactly replaced by cards.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,16 +1,21 @@
 @import 'bootstrap';
```
Comment (mejackreed) [inline app/assets/stylesheets/application.scss]: Have you considered changing the import order here? 

Perhaps this order could let us define BS variables before they are evaluated?
```sass
@import 'sul-variables';
@import 'bootstrap-variables';
@import 'bootstrap';
```

File: app/assets/stylesheets/bootstrap-overrides.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.panel {
```
[the remark this answers is not in the document]
Comment (camillevilla) [reply app/assets/stylesheets/bootstrap-overrides.scss]: alrighty, I've removed these 👍

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (camillevilla) [reply app/assets/stylesheets/application.scss]: ✅ it works!

**Comment by camillevilla** [comment]
@mejackreed I've made requested changes and did some more Bootstrap 3 --> 4 translation in `bootstrap-variables.scss`.

**Pull request merged**